src/ShallowWrapper.js

```diff
@@ -3,6 +3,7 @@ import flatten from 'lodash/flatten';
 import unique from 'lodash/uniq';
 import compact from 'lodash/compact';
 import cheerio from 'cheerio';
+import merge from 'lodash/merge';
```

Comment:
deep merging the options sounds like too much, i think eg merging the context as in your example should be done manually by the user. thoughts?